docs/agents/ucsc_radiometer.rst

```diff
@@ -0,0 +1,70 @@
+.. highlight:: rst
+
+.. _ucsc_radiometer:
+
+==============
+UCSC Radiometer Agent
+==============
+
+The UCSC Radiometer agent uses HTTP queries to publish pwv data from a Flask server.
+
+.. argparse::
+    :filename: ../socs/agents/ucsc_radiometer/agent.py
+    :func: add_agent_args
+    :prog: python3 agent.py
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
```

Comment:
Conventionally (and consistent with the rest of the document) there should be a blank line after the `------` and before the paragraph here.